Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -23,7 +23,7 @@
     public class RequestTrackingTelemetryModule : ITelemetryModule, IDisposable
     {
         private readonly EventListener listener;
-
+        private readonly IList<FilterRequest> handlersToFilter = new List<FilterRequest>();
```

Comment:
I had it there: https://mseng.visualstudio.com/DefaultCollection/AppInsights/_git/AppDataCollection/pullrequest/59962?view=compare&path=%2FWeb%2FWeb%2FWeb.Shared.Net%2FRequestTrackingTelemetryModule.cs and I tested it for sure - let me find out